utils/custom.js

```diff
@@ -111,7 +111,7 @@ const handleAnonymousData = async (
   let childComment = data.latest_children?.comment || [];
   childComment = await Promise.all(
     childComment.map(async (child) => {
-      const user = karmaScores.find((user) => user.user_id === child.data.user_id);
+      const user = karmaScores.find((user) => user.user_id === child.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `child.data.user_id` to `child.user_id` might cause issues if the `user_id` is not directly available on the `child` object but nested inside the `data` property. Please ensure that the `user_id` is always directly accessible on the `child` object, otherwise this could lead to undefined behavior or errors.
```diff
- const user = karmaScores.find((user) => user.user_id === child.user_id);
+ const user = karmaScores.find((user) => user.user_id === child.data?.user_id);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->